Source/spelldat.h

```diff
@@ -217,6 +217,6 @@ struct SpellData {
 	int sStaffCost;
 };
 
-extern SpellData spelldata[];
+const extern SpellData spelldata[];

```

Comment:
Only a nit: in the codebase only `extern const` is used and never `const extern`